CHANGELOG.md

```diff
@@ -2,7 +2,35 @@
 
 ## Next version
 
-- Put your changes here...
+- Replaced the vnu-jar based HTML validator with express-html-validator 🎉.
```

Comment:
merging this is blocked by https://github.com/rooseveltframework/express-html-validator/pull/7

Reply:
good to go now, update the dep and regenerate pkg lock